blockreassurance.php

```diff
@@ -47,49 +47,19 @@ class blockreassurance extends Module implements WidgetInterface
     const POSITION_BELOW_HEADER = 1;
     const POSITION_ABOVE_HEADER = 2;
 
-    /** @var string */
-    public $name;
-    /** @var string */
-    public $version;
-    /** @var string */
-    public $author;
-    /** @var bool */
-    public $need_instance;
```

Comment:
I think this one has been added for compatibility reason ^^

Reply:
`$need_instance` ? It was already inside 1.5 version of PrestaShop.

Reply:
I don't remember why I said that :sweat_smile:

Reply:
Its maybe why we close Pull Request after 30 days, 🤣

Reply:
Oh, I remember, it's because of this:
https://github.com/PrestaShop/blockreassurance/runs/4090566413?check_suite_focus=true#step:7:219

Reply:
In the Core, we except an int, whereas it should be a boolean

Reply:
Okay. So the issue is more inside this: https://github.com/PrestaShop/blockreassurance/pull/305/files#diff-8473374a31fcc2071f982bbe46a4f5d279c23ed5e4acdb0471cefeaf9121b7b8R74